src/endpoints/seed/pages/page.ts

```diff
@@ -76,9 +76,9 @@ export const page: (
     },
     layout: [
       {
+        color: 'white',
         columns: [
           {
-            size: 'full',
```

Comment:
@rchlfryn this looks like Content > Layout 1 > Content > RichText - maybe this is the error?

Reply:
```
[18:39:14] INFO: Creating pages['nwac']['donate-membership'] with hash 39402c69...
[18:39:14] ERROR: The following field is invalid: Content > Layout 1 > Content > RichText
    err: {
      "type": "ValidationError",
      "message": "The following field is invalid: Content > Layout 1 > Content > RichText",
      "stack":
          ValidationError: The following field is invalid: Content > Layout 1 > Content > RichText
```

Reply:
Wait no, rich text is after this diff and has no changes ...